addons/html_editor/static/src/fields/html_viewer.js

```diff
@@ -221,31 +218,26 @@ export class HtmlViewer extends Component {
             env,
             props,
         });
-        const app = new App(Component, {
-            test: dev,
-            env,
-            translateFn,
-            getTemplate: getRawTemplate,
+        const app = this.__owl__.app;
+        const root = app.createRoot(Component, {
             props,
+            env,
         });
-        app.rawTemplates = mainApp.rawTemplates;
-        // Can't copy compiled templates because they have a reference to the main app
-        // app.templates = mainApp.templates;
-        const promise = app.mount(host);
+        const promise = root.mount(host);
         // Don't show mounting errors as they will happen often when the host
         // is disconnected from the DOM because of a patch
         promise.catch();
         // Patch mount fiber to hook into the exact call stack where app is
         // mounted (but before). This will remove host children synchronously
         // just before adding the app rendered html.
```

Comment:
Same as for the other PR, the comments could be updated to accommodate the new root structure.